src/lib/account/account.ts

```diff
@@ -53,5 +54,16 @@ export const deriveAccountIdsAndMetadata: DeriveAccountIdsAndMetadata = async ar
       }),
     ),
   )
-  return merge({}, ...result)
+
+  const fulfilledAccountIdsAndMetadata = settledAccountIdsAndMetadata
+    .filter(result => {
+      if (isRejected(result)) {
+        console.error(result.reason)
```

Comment:
this log useful?

Reply:
Might as well? For demo wallet, that will obviously be fluff but for other wallets, could be a legit error